decentraland/explorer
# feat: Implement BiW exit modal

When you click on the exit button of the BiW editor, the player should be prompted with an exit modal to confirm that they really want to exit.

![image.png](https://images.zenhubusercontent.com/603cdb58edb53386a3d0d8d5/7f34ddaf-44ce-4588-a095-6934f226f691)

### Testing
1. Go to: https://explorer.decentraland.zone/branch/feat/Implement-BiW-exit-modal/index.html?ENV=org&ENABLE_BUILDER_IN_WORLD&position=100,100&realm=fenrir-amber
2. Activate the Builder In World editor pressing the "**K**" key.
3. Click on the exit button.
4. See the new confirmation modal.
